packages/story-editor/src/utils/getInUseFonts.ts

```diff
@@ -40,23 +44,31 @@ export function getInUseFontsForPages(pages) {
   );
 }
 
+interface getTextSetsForFontsProps {
+  fonts: string[];
+  textSets: TextSet[];
+}
+
 /**
  * Returns a list of text sets that include the fonts provided in the options object.
  *
- * @param {Array<Object>} options Object containing an array of fonts and text sets to filter.
- * @return {Array<Object>} List of text sets that match the array of fonts.
+ * @param options Object containing an array of fonts and text sets to filter.
+ * @return List of text sets that match the array of fonts.
  */
-export function getTextSetsForFonts({ fonts, textSets }) {
+export function getTextSetsForFonts({
+  fonts,
+  textSets,
+}: getTextSetsForFontsProps) {
   return textSets
     .map((currentTextSet) => {
       const hasFontInUse = currentTextSet.elements.reduce(
         (elementMemo, currentElement) => {
           if (
-            currentElement.type === 'text' &&
+            elementIs.text(currentElement) &&
             Boolean(currentElement.font?.family)
```

Comment:
Since `font.family` is required in a text element then this additional check shouldn't be necessary.

Reply:
Fixed.